build_system/src/test.rs

```diff
@@ -840,6 +840,7 @@ fn should_remove_test(file_path: &Path) -> Result<bool, String> {
             "-Cllvm-args",
             "//~",
             "thread",
+            "//@ known-bug"
```

Comment:
Actually, I was wrong on this one as it seems some of those tests can pass.
Please remove it and add the tests to `failing-ui-tests.txt`.
Sorry.

Reply:
Ahhh! yes, I missed this as well.